selfdrive/ui/sunnypilot/qt/offroad/settings/vehicle_panel.cc

```diff
@@ -20,11 +20,101 @@ VehiclePanel::VehiclePanel(QWidget *parent) : QFrame(parent) {
   platformSelector = new PlatformSelector();
   list->addItem(platformSelector);
 
+//   hkgtuningToggle = new ParamControl("HKGtuning", (add all toggle parameters here), this);
+
+  QObject::connect(uiState(), &UIState::offroadTransition, [=](bool offroad) {
+    isOnroad = !offroad;
+    updateCarToggles();
+  });
+
   ScrollViewSP *scroller = new ScrollViewSP(list, this);
   vlayout->addWidget(scroller);
 
   QObject::connect(uiState(), &UIState::offroadTransition, platformSelector, &PlatformSelector::refresh);
```

Comment:
L33 already connects `UIState::offroadTransition` to `PlatformSelector::refresh`, so we can consolidate the 2 `QObject::connect` together.

Let's keep using `offroad` instead of `isOnroad`, so it's more consistent with what we already use for `DeveloperPanel` and `PlatformSelector`.